docs/setup/windows.md

```diff
@@ -18,7 +18,7 @@ MetaDescription: Get Visual Studio Code up and running on Windows
 
 You can also find a Zip archive [here](/docs/?dv=winzip).
 
->**Note:** .NET Framework 4.5 is required for VS Code.  If you are using Windows 7, please make sure [.NET Framework 4.5](https://www.microsoft.com/en-us/download/details.aspx?id=30653) is installed.
+>**Note:** .NET Framework 4.5.2 is required for VS Code.  If you are using Windows 7, please make sure [.NET Framework 4.5.2](https://www.microsoft.com/en-us/download/details.aspx?id=42642) is installed.
```

Comment:
Any reason you choose to link to the offline installer rather than the web installer at https://www.microsoft.com/en-us/download/details.aspx?id=42643.  The 4.5 link was for the web installer which is probably useable for the majority of users.